<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration_window</class>
 <widget class="QDialog" name="registration_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px solid rgba(0, 0, 0, 0);
border-radius: 7px;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>400</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="labelMain">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSecond">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Придумайте данные для входа</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterRegistrationLogin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
font-size: 20px;
padding-left: 10px;
color: rgb(130, 130, 130);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Пользователь</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterRegistrationPassword">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
font-size: 20px;
padding-left: 10px;
color: rgb(130, 130, 130);</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="placeholderText">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonRegistration">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
font-size: 16px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
}

QPushButton:hover {
background-color: rgba(0, 0, 0, 95);
}

QPushButton:pressed {
background-color: rgba(0, 0, 0, 85);
}
</string>
    </property>
    <property name="text">
     <string>Зарегистрироваться</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelinfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>361</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
color: rgb(130, 130, 130);</string>
    </property>
    <property name="text">
     <string>Перед завершением регистрации убедитесь, что</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelinfo2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
color: rgb(130, 130, 130);</string>
    </property>
    <property name="text">
     <string>вы запомнили данные для входа</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>628</y>
     <width>1281</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res-rc.qrc">:/icons/logo/faq.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>650</y>
     <width>63</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>FAQ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1105</x>
     <y>660</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
color: rgb(130, 130, 130);</string>
   </property>
   <property name="text">
    <string>© 2024 “Mirage”</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>255</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="labelLogo">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res-rc.qrc">:/icons/logo/logopen72.svg</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelLogoText">
      <property name="styleSheet">
       <string notr="true">font-size: 20px;

font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Equipment Tracker</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res-rc.qrc"/>
  <include location="venv/Lib/site-packages/PySide6/res-rc.qrc"/>
 </resources>
 <connections/>
</ui>
